packages/collections/persistent/_test.pony

```diff
@@ -75,11 +69,11 @@ class iso _TestListApply is UnitTest
     h.assert_eq[U32](l1(0), 1)
     h.assert_eq[U32](l1(1), 2)
     h.assert_eq[U32](l1(2), 3)
-    h.assert_error({()? => l1(3) })
-    h.assert_error({()? => l1(4) })
+    h.assert_error({()? => l1(3)})
+    h.assert_error({()? => l1(4)})
 
     let l2 = Lists[U32].empty()
-    h.assert_error({()? => l2(0) })
```

Comment:
I don't know if we have formally established a stdlib style for these yet, but I prefer to see this space before the end of the lambda body remain, as it mirrors the space that comes before the lambda body (after the fat arrow).